### DISCUSSION TITLE
Hise rows for users

### DISCUSSION THREAD

**Discussion by AlexWirthAAU** [grafana/grafana#46786] [Dashboards, Panels & Visualizations]
Hiding specific rows within a dashboard for some of the users that can access the dashboard would be very helpful. I have seen some issues where this was asked and it seems like this is not possible yet.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
